packages/components/src/ui/cardCarousel.tsx

```diff
@@ -0,0 +1,63 @@
+import { Card, CardContent } from "@alittlebyte/components/ui/card"
+import {
+	Carousel,
+	CarouselContent,
+	CarouselItem,
+	CarouselPrevious,
+	CarouselNext,
+} from "@alittlebyte/components/ui/carousel"
+import { FC } from "react"
+
+type CardCarouselProps = {
+	data: [
+		{
+			title: string
+			description: string
+			image: string
+		},
+	]
```

Comment:
I had never seen that before. But the right way is like that
```suggestion
	data: {
			title: string
			description: string
			image: string
		}[]
```